src/store/movie/slice/index.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from "@reduxjs/toolkit";
+import * as updateStateFunctions from "./update-state-functions";
+
+const movieSlice = createSlice({
+    name: "movies",
+    initialState: {
+        movies: [],
+        selectedMovie: {
+            id: localStorage.getItem("selectedMovieId"),
+            title: localStorage.getItem("selectedMovieTitle"),
+            description: localStorage.getItem("selectedMovieDescription"),
+            imageUrl: localStorage.getItem("selectedMovieImageUrl"),
+            genre: localStorage.getItem("selectedMovieGenre"),
+            num_visits: localStorage.getItem("selectedMovieVisits"),
+            likes: localStorage.getItem("selectedMovieLikes"),
+            dislikes: localStorage.getItem("selectedMovieDislikes"),
+        },
```

Comment:
Pre pre komplikovano, `selectedMovie` bas to treba i da bude - da stavis kompletan movie objekat i onda da vadis sta ti je potrebno.

Reply:
Reseno, i ovo i sve slicno.